stag_node:
  ros__parameters:
    # STag parameters
    libraryHD: 15
    errorCorrection: 7
    # Camera configuration
    camera_info_topic: /usb_cam/camera_info
    raw_image_topic: /usb_cam/image_raw
    is_compressed: false
    # Node configuration
    tag_tf_prefix: STag_
    show_markers: true
    publish_tf: true
